### ISSUE TITLE
Missing HTML file hangs JavaFX

### ISSUE THREAD

**Issue by tresf** [qzind/tray#412]
When a URL is provided to HTML printing, the error is detected but never recovered.

QZ Tray 2.1.0 (HEAD)
```
[ERROR] 2019-02-05 11:12:40,417 @ qz.utils.PrintingUtilities:249
	Failed to print
java.awt.print.PrinterException: File not found
	at qz.printer.action.PrintHTML.print(PrintHTML.java:259)
```

![image](https://user-images.githubusercontent.com/6345473/52287012-58b97500-2937-11e9-80b7-44939811ad6f.png)

**Comment by bberenz** [comment]
So the empty window showing is because of an old stop-gap being called to ensure it's rendering content correctly at the start of a print, but this exception is happening before the "load" lambda gets called (where we'd catch and hide it on an exception _during_ a print).

I can't seem to replicate the issue the stop-gap was put in to fix when taken out, so it seems like some of the other changes we've made since then have managed to take care of the issue. This unbreaks headless mode for html prints and also prevents this issue from happening in the first place. So I'm going to cautiously say eaa2b658469db2b42d30de357cf8db00a58207dd fixes this and doesn't break other things.

**Comment by tresf** [comment]
>  it seems like some of the other changes we've made since then have managed to take care of the issue

Could this be due to switching to Gluon's JavaFX 11?  If we reintroduce the bug for Java 8, we may need this stopgap in a conditional.  We'll test 2.1 against Java 8 and see.

**Comment by lite1979** [comment]
Tested latest 2.1 against Oracle Java 8 Update 211 in Windows 10; no preview window, no errors for HTML print. Tested to XPS Document Writer and Epson TM-T88V

**Comment by tresf** [comment]
Closed via eaa2b65.
